src/utils/default-plugin.ts

```diff
@@ -1,14 +1,46 @@
 import { InputOptions, Plugin } from '../rollup/types';
 import error from './error';
-import { lstatSync, readdirSync, readFileSync, realpathSync } from './fs'; // eslint-disable-line
+import { lstatSync, readdirSync, readFile, realpathSync, stat } from './fs'; // eslint-disable-line
 import { basename, dirname, isAbsolute, resolve } from './path';
 
+interface LoadMemoization {
+	[key: string]: {
+		mtimeMs: number;
+		size: number;
+		contents: string;
+	};
+}
+
 export function getRollupDefaultPlugin(options: InputOptions): Plugin {
+	const loadMemoization: LoadMemoization = {};
 	return {
 		name: 'Rollup Core',
 		resolveId: createResolveId(options),
 		load(id) {
-			return readFileSync(id, 'utf-8');
+			if (loadMemoization[id]) {
```

Comment:
If the "if" has no "else", "load" will only ever return `undefined`. Which it did for me when I attempted to try this out. This is also why most tests are red.